solutions/standard/variables.tf

```diff
@@ -0,0 +1,297 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API key."
+  sensitive   = true
+}
+
+variable "prefix" {
+  type        = string
+  description = "The prefix for resources created by this solution."
```

Comment:
Can you use same as this: https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/fbfe71b4705e422018300bdecd37d5f99f63f244/solutions/standard/variables.tf#L38-L42

It should have a default value